'Leather' theme

ryujinscales opened the pull request:
![image](https://user-images.githubusercontent.com/97795016/152107019-ae151ac7-f81a-44af-9af6-f19ec3de0070.png)
above : flip test colors and colorful mode off

![image](https://user-images.githubusercontent.com/97795016/152107093-51ac4ff7-77ea-4132-b0a1-549e1cc14a3e.png)
above :  flip test colors on

![image](https://user-images.githubusercontent.com/97795016/152107150-8b7e90b6-4c3e-4447-9547-28ab0e52ffe9.png)
above ; flip test colors and colorful mode on

![image](https://user-images.githubusercontent.com/97795016/152107214-229ffc91-e644-4a31-9c52-0297137ce64a.png)
above : colorful mode on



Hope it get's approved !
Signing off, ryujinscales

rizwanmustafa commented:
Remove the greek fire theme cz it has already been rejected.

rizwanmustafa commented on static/themes/greek_fire.css:
```diff
@@ -0,0 +1,11 @@
+:root {
```
Remove this file.

rizwanmustafa commented on static/themes/_list.json:
```diff
@@ -699,4 +704,14 @@
     "bgColor": "linear-gradient(to right, #002c39, #015061, #007a7f, #00e980, #00ffbb)",
     "mainColor": "#ffffff"
   }
+  ,{
```
Remove the greek_fire entry

ryujinscales replied:
sure

rizwanmustafa commented on static/themes/ryujinscales.css:
```diff
@@ -0,0 +1,13 @@
+:root {
```
This file is the same as the greek theme, so remove it.

ryujinscales replied:
bruh the ryujinscales theme has been approved

rizwanmustafa replied:
It is literally the same as the greek fire theme. If the greek fire theme was rejected, how can this be approved. And if this was approved, can you provide me the pr link.

ryujinscales replied:
https://github.com/Miodec/monkeytype/pull/2368

ryujinscales replied:
see for urself
then consult me

rizwanmustafa replied:
Bro, if the theme was already merged, why are you adding it again. Remove this file as the ryujinscales theme has already been added.

rizwanmustafa replied:
And from what I believe, this file is the culprit of merge conflicts.

ryujinscales replied:
> The ryujinscales theme has already been added and merged into Monkeytype, so please remove this file.

which file
send the link
im not sure

ryujinscales replied:
r u sure it wont delete the theme

Miodec replied:
Since the theme has been already merged in it will not delete it

ryujinscales commented:
Pull_request by ryujinscales · Pull Request #2368 · Miodec/monkeytype
(github.com) <https://github.com/Miodec/monkeytype/pull/2368>

On Wed, Feb 2, 2022 at 1:06 PM Rizwan Mustafa ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In static/themes/ryujinscales.css
> <https://github.com/Miodec/monkeytype/pull/2396#discussion_r797335045>:
>
> > @@ -0,0 +1,13 @@
> +:root {
>
> It is literally the same as the greek fire theme. If the greek fire theme
> was rejected, how can this be approved. And if this was approved, can you
> provide me the pr link.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Miodec/monkeytype/pull/2396#discussion_r797335045>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXKDXSHI47FAZ4FC3LEFRITUZDNGPANCNFSM5NLGP5KA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

rizwanmustafa commented:
@ryujinscales  https://github.com/Miodec/monkeytype/blob/master/static/themes/ryujinscales.css

ryujinscales commented:
> @ryujinscales https://github.com/Miodec/monkeytype/blob/master/static/themes/ryujinscales.css

#2397

rizwanmustafa commented:
What I meant by sending the file link was that the ryujinsacales theme is already added so delete the file in this pr and not the project as a whole.

Miodec commented:
I recommend you fetch all changes from upstream and merge to get rid of the conflicts.

Miodec commented:
As for the theme itself, please include some screenshots when not typing too.

Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.

ryujinscales commented:
> I recommend you fetch all changes from upstream and merge to get rid of the conflicts.

and how exactly do i do that

ryujinscales commented:
> As for the theme itself, please include some screenshots when not typing too.
> 
> Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.

i updated it

ryujinscales commented:
> > As for the theme itself, please include some screenshots when not typing too.
> > Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.
> 
> i updated it

updated bgcolor

![image](https://user-images.githubusercontent.com/97795016/152186960-5a7083bf-0177-4b89-9321-9ff439962ac6.png)

Miodec commented:
How about this instead
![image](https://user-images.githubusercontent.com/13181393/152203969-c14f7690-18db-4980-8a05-5a7b101a086a.png)
